src/main/java/com/mercadopago/client/order/OrderRefundRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.mercadopago.client.order;
+
+import lombok.Builder;
+import lombok.Getter;
+import java.util.List;
+
+@Builder
+@Getter
+/** OrderRefundRequest class. */
+public class OrderRefundRequest {
+
+        private List<OrderRefundPaymentRequest> transactions;
+
+}
```

Comment:
Ajustar e adicionar comentário da propriedade
```suggestion
/** OrderRefundRequest class. */
@Builder
@Getter
public class OrderRefundRequest {

        private List<OrderRefundPaymentRequest> transactions;
}
```